.github/workflows/Build.yml

```diff
@@ -7,24 +7,23 @@ on:
   pull_request:
     branches: [ main ]
 
+env:
+  mod_name: Creepers
```

Comment:
Just realized the issue with this, if there's gonna be a separate Releases.yml for auto releases, then this will have to be defined there as well, which kinda defeats the purpose of not using the metadata. 🤔 Might as well remove the variable usage entirely and just upload the artifact hardcoded as `Creepers`

Reply:
Not super familiar with Actions yet, is there some sort of common environment it can pull from across different jobs?

Reply:
Found this but it uses a marketplace job: https://arinco.com.au/blog/github-actions-share-environment-variables-across-workflows/

As a code purist I can't say I'm excited to implement it, but it's up to you if it's worth the hassle. Also just as easy to hardcode or maybe even pull from the repo name

Reply:
Okay based on the actions docs there is `$GITHUB_REPOSITORY` which will evaluate to `Caaz/rimworld-creepers` here. So possibly could do something like:
```console
$ MOD_NAME=$(mod_name=${GITHUB_REPOSITORY##*-}; echo ${mod_name^})
$ echo $MOD_NAME
Creepers
```

Reply:
Well right now the job reads from the metadata.json which has the mod name in it as well, and the release job _could_ do that too, if we wanna keep variables.

Reply:
both the Release and the Build job reading from the metadata.json feels like the better option, if not a little repeaty.

Reply:
In that case I could revert https://github.com/Caaz/rimworld-creepers/pull/9/commits/445d6172a26b57ca76554c8e7efffa62c027ce71 if that feels best

Reply:
yeh

Reply:
Reverted, combined run call retained